### ISSUE TITLE
[Improve] - Add prettier to the project

### ISSUE THREAD

**Issue by gurpreet-legend** [gurpreet-legend/Alan-Feed#5]
### Improvements:
- [ ] Add `prettier` to this project to make the code formatting easy fro other developers
- [ ] Add relevant scripts to `package.json` for easy access to prettier commands

Here are steps to add prettier to Node projects, you can refer this article:
https://blog.bitsrc.io/add-prettier-to-your-project-d7e91ac03d05

**Comment by Harshwardhan-Patil** [comment]
hey @gurpreet-legend , I'd like to work on this. Assign me

**Comment by gurpreet-legend** [comment]
Sure, you can work on this @Harshwardhan-Patil :)

**Comment by Harshwardhan-Patil** [comment]
Hey @gurpreet-legend ,I am getting problem while installing npm packages its giving error like **INVALID VERSION**.
Is there problem in version value in **package.json** file? Can you plz help me?

<img width="960" alt="Screenshot (89)" src="https://user-images.githubusercontent.com/100856982/196952702-74cbe470-3c2b-4aa3-9819-5aef5eb6dbf7.png">

**Comment by gurpreet-legend** [comment]
@Harshwardhan-Patil, this should not happen, can you change the version to `1.0.0` and check if it installs all packages or not.

**Comment by Harshwardhan-Patil** [comment]
@gurpreet-legend ,this is not working and what is  **ansi-html-community**
<img width="960" alt="Screenshot (90)" src="https://user-images.githubusercontent.com/100856982/197207858-d445e496-ab9e-4487-998b-56215f04871b.png">

**Comment by gurpreet-legend** [comment]
@Harshwardhan-Patil , try deleting the `package-lock.json` and reinstalling it, it might work

And `ansi-html-community` is a dependency library for `alan-ai`, I read about it, it converts ANSI text to HTML, something alan-ai needs internally to work, even if you remove it and try installing deps even then alan-ai will ask you to download it.

**Comment by Harshwardhan-Patil** [comment]
@gurpreet-legend ,It works.Thank you for helping me

**Comment by Harshwardhan-Patil** [comment]
Hey @gurpreet-legend ,I have solved this issue and created pull request.Plz review my PR and tell me if any changes you want.

**Comment by Harshwardhan-Patil** [comment]
Hey @gurpreet-legend , 
1.  Should  I add **.prettierignore** file if yes which file you want to ignore me  
2.  I check prettier install documentation they say "**If you use ESLint, install [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier#installation) to make ESLint and Prettier play nice with each other [Docs](https://prettier.io/docs/en/install.html#eslint-and-other-linters)**"

**Comment by gurpreet-legend** [comment]
Yes, you can do that as well, adding the prettier plugin with eslint also works fine. Also make sure to pull the recent changes before pushing your code :)

**Comment by Harshwardhan-Patil** [comment]
Which file/folder should I add  to  .prettierignore

**Comment by Harshwardhan-Patil** [comment]
Hey @gurpreet-legend , I have created the pull request but it giving me Merge Conflicts ,can you please help me resolving those issue ?

**Comment by Harshwardhan-Patil** [comment]
Hey @gurpreet-legend ,I've created the pull request, but it gives me merge conflicts. Can you help me resolve those issues?
